system_test/helpers/aest_nodes.erl

```diff
@@ -346,16 +348,25 @@ get_block(NodeName, Height) ->
     end.
 
 get_top(NodeName) ->
-    {ok, 200, Top} = request(NodeName, 'GetTop', #{}),
+    {ok, 200, Top} = request(NodeName, 'GetTopBlock', #{}),
     Top.
 
+get_mempool(NodeName) ->
+    {ok, 200, Txs} = request(NodeName, 'GetPendingTransactions', #{}),
+    case {maps:get(<<"transactions">>, Txs, undefined),
+          maps:get(transactions, Txs, undefined)} of
+        {undefined, Mempool} -> Mempool; %% future proof
```

Comment:
Is this for upcoming API changes in PRs 4/5 or 5/5?

Reply:
Yes.